app/page.tsx

```diff
@@ -1,39 +1,126 @@
-import Link from "next/link"
+"use client"
 
-import { siteConfig } from "@/config/site"
-import { buttonVariants } from "@/components/ui/button"
+import React, { useEffect, useState } from "react"
```

Comment:
No need to import React explicitly.

https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html
<img width="839" alt="image" src="https://github.com/user-attachments/assets/6ee2cacb-5fc3-47d3-8e0c-250ef2ca81e1">